docker/centos-graal-maven-s2i/README.md

```diff
@@ -0,0 +1,30 @@
+# [Quarkus.io](http://quarkus.io) native S2I
+
+## How to build this S2I builder image
+
+### OpenShift
+
+    oc new-build https://github.com/quarkusio/quarkus.git --context-dir=docker/centos-graal-maven-s2i --name quarkus-native-s2i
+
+### Locally
+
+    docker build . -t quarkus-native-s2i
+
+## How to use this
+
+### OpenShift
+
+    oc new-app quarkus-native-s2i~https://github.com/quarkusio/quarkus-quickstarts --context-dir=getting-started-native --name=getting-started-native
+
+Note that GraalVM-based native build are more memory & CPU intensive than regular pure Java builds,
+and you therefore may need to increase the quota for OpenShift's S2I builder containers.
```

Comment:
Can you give some hint about this? 
I guess you can edit the build object.

Reply:
I was hoping someone else would help with how exactly to do that... :smiley:

Reply:
There now is at least some doc. Could always be further improved, but god enough for a 0.1?